app/controllers/application_controller.rb

```diff
@@ -11,6 +11,7 @@ def authenticate!
     respond_to do |format|
       format.html { redirect_to root_path }
       format.json { unauthorized_json_access }
+      format.pdf { redirect_to root_path }
```

Comment:
No need to add `Mime::Type.register "application/pdf", :pdf` under `config/initializers/mime_types.rb`?

Reply:
No, the 'applicaion/pdf' mime type is already register in Rails 5 :smiley: